caf-audit-service-container/pom.xml

```diff
@@ -339,7 +337,7 @@
                                 <env>
                                     <CAF_APPNAME>test/auditservice</CAF_APPNAME>
                                     <CAF_CONFIG_PATH>/config</CAF_CONFIG_PATH>
-                                    <CAF_ELASTIC_HOST_AND_PORT>${docker.host.address}:${elasticsearch.transport.port}</CAF_ELASTIC_HOST_AND_PORT>
+                                    <CAF_ELASTIC_HOST_AND_PORT>elasticsearch:9300</CAF_ELASTIC_HOST_AND_PORT>
```

Comment:
Yes I have configured private networking between the webservice and elasticsearch instances with the <links> element.